components/pd_client/src/util.rs

```diff
@@ -312,7 +312,7 @@ impl Client {
         F: FnMut(&Client, Req) -> PdFuture<Resp> + Send + 'static,
     {
         Request {
-            remain_reconnect_count: retry,
+            remain_request_count: retry,
```

Comment:
I think it should be remain_reconnect_count?

Reply:
I change the behavior, it indicate the total count of the request.

Reply:
Then, the `retry` should also be changed?

Reply:
It only changed the retries from 13 times to 10 times. before the first 3 times do not count as `reconnect_count`, but the current is.